### PR TITLE
xd75-helpful keymap

### PR DESCRIPTION
**Pull request by amesa0**
Keymap that is similar to typical 60% boards.

I used macro to open the terminal and have the function with a keymap that is useful for command line operations.

TODO:

### CODE REVIEW

**Comment by jackhumbert** [comment]
Cool! Would you mind making the keymap name lowercase? We try to keep them all consistent for case-sensitive systems :)

**Comment by amesa0** [comment]
I have tried again. Can you share with me the error now? 
@jackhumbert

**Comment by drashna** [comment]
Looks like some of the keymaps didn't get renamed (the xd75)

**Comment by amesa0** [comment]
I believe there is some confusion, I have changed the name for xd75 keymap from 'Helpful' to 'helpful' during last commit. 

Is there another naming issue?

**Comment by drashna** [comment]
That appears to have worked!

**Comment by amesa0** [comment]
YES!!!!!!!

**Comment by jackhumbert** [comment]
Nice! Would you mind replacing `REPLACE_WITH_YOUR_NAME` with your name if you're the author of these files?

**Comment by amesa0** [comment]
I am trying to find where I should be replacing `REPLACE_WITH_YOUR_NAME` however I cannot find it. Can you direct me @drashna (sorry for the trouble)

File: keyboards/30sqrt2/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# 30sqrt2
```
[the remark this answers is not in the document]
Comment (amesa0) [reply keyboards/30sqrt2/readme.md]: Done I hope. I am glad I have had all of these issues in my first pull so I will not have them later.

Also @drashna I have some other questions regarding QMK and what I can add to the files apart from Keymaps. Can I PM you on Reddit if you are there or better to talk discord or elsewhere?

**Comment by amesa0** [comment]
wait what did I do?

**Comment by drashna** [comment]
You closed the PR, it looks like.

File: keyboards/30sqrt2/keymaps/default/config.h [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/30sqrt2/keymaps/default/config.h]: This should be `#include QMK_KEYBOARD_CONFIG_H`

File: keyboards/30sqrt2/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "30sqrt2.h"
```
Comment (drashna) [inline keyboards/30sqrt2/keymaps/default/keymap.c]: This should be `#include QMK_KEYBOARD_H`

File: keyboards/30sqrt2/readme.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (drashna) [reply keyboards/30sqrt2/readme.md]: Sorry, missed this.  The main "conversation" page should be fine.  But you can PM me on reddit.  There is also the gitter page for QMK: https://gitter.im/qmk/qmk_firmware

**Comment by jackhumbert** [comment]
@amesa0 just wanted to check to see if you could make the changes that @drashna mentioned here. Thanks!

**Pull request closed without merging**